src/junox/lfo.js

```diff
@@ -1,18 +1,46 @@
 export default class LFO {
-  constructor ({ frequency, sampleRate }) {
-    this.frequency = frequency
-    this.sampleRate = sampleRate
+  constructor ({ frequency, delay = 0, sampleRate }) {
+    this.anchorRate = 1 / sampleRate
     this.phase = 0
-    this.tick()
+    this.setRate(frequency)
+    this.setDelay(delay)
+    this.trigger()
   }
 
-  tick () {
-    this.samplesPerPeriod = Math.round(this.sampleRate / this.frequency)
-    this.phase = (this.phase + 1) % this.samplesPerPeriod
+  setRate (frequency) {
+    this.incrementRate = this.anchorRate * frequency
+  }
+
+  setDelay (delay) {
+    this.delay = delay * this.anchorRate
+  }
+
+  trigger () {
+    this.delayPhase = 0
+  }
+
+  delayEnv () {
+    this.delayPhase += this.delay
+    if (this.delay < 0.0001 || this.delayPhase > 2) {
+      return 1
+    } else if (this.delayPhase < 1) {
+      return 0
+    }
+    // from juno manual the volume seems to grows by the same distance it's delayed
```

Comment:
Ideally this this should be curved.  Normally attack is modelled as "newValue = (1-oldValue)*rateFactor" ... so starts linear-ish then flattens out as it gets closer to full volume.

Warning: Pictures in manual are often simplified.